src/problem_bank_scripts/problem_bank_scripts.py

```diff
@@ -1271,6 +1271,10 @@ def process_question_pl(source_filepath, output_path=None, dev=False):
     # Fix Latex underscore bug (_ being replaced with \_)
     question_html = question_html.replace("\\_", "_")
 
+    # Fix Latex bracket bug ([ and ] replaced with \[ and \])
+    # See https://github.com/open-resources/problem_bank_scripts/issues/89
+    question_html = question_html.replace("\\[","[").replace("\\]","]")
+
```

Comment:
Following up on this since its more relevant again

I think this should be able to resolve #88 and #89 while also simplifying the code, and handling all of the cases we identified
```suggestion
    # Fix Latex over-escaping from mdformat (i.e. _, [, and ]being replaced with \_, \[, and \])
    # See https://github.com/open-resources/problem_bank_scripts/issues/89
    # Also see https://github.com/open-resources/problem_bank_scripts/pull/92
    question_html = re.sub(r"\\\\([\[\]_`\\\*<])", r"\1", question_html, re.MULTILINE)

```
Alternatively, a regex-free solution:
```suggestion
    # Fix Latex over-escaping from mdformat (i.e. _, [, and ]being replaced with \_, \[, and \])
    # See https://github.com/open-resources/problem_bank_scripts/issues/89
    # Also see https://github.com/open-resources/problem_bank_scripts/pull/92
    question_html = question_html.replace("\\_", "_").replace("\\[","[").replace("\\]","]")
    question_html = question_html.replace("\\*", "*").replace("\\<","<").replace("\\`","`")

```